src/setup-ocaml/installer.ts

```diff
@@ -142,6 +143,7 @@ export async function installer(): Promise<void> {
     core.exportVariable("DUNE_CACHE", "enabled");
     core.exportVariable("DUNE_CACHE_TRANSPORT", "direct");
   }
+  core.exportVariable("CLICOLOR_FORCE", CLICOLOR_FORCE);
```

Comment:
Why not just add "1" here?

```ts
  core.exportVariable("CLICOLOR_FORCE", "1");
```

Reply:
I thought I was re-exporting the value from `contants.ts`. Isn't it possible to set the value only once? If not, I'm ok with your change.